typescript-template-literals/main.ts

```diff
@@ -0,0 +1,29 @@
+const number = 9200;
```

Comment:
Starting here update this ts ( typescript ) file from js to modern ts. Then resubmitt.
**Remember** if it isn't specified still do so as it is a requirement for a correct submission.